templates/bake/Model/table.twig

```diff
@@ -81,6 +77,7 @@ class {{ name }}Table extends Table
 {{- "\n" }}
 
 {%- if validation %}
+{% set generatedFunctions = generatedFunctions|merge(['validationDefalt']) %}
```

Comment:
Typo in `validationDefalt`, should have been caught by a test case.

Reply:
Will need many more tests once we decide on the template changes.